docs/semgrep-secrets/policies.md

```diff
@@ -0,0 +1,123 @@
+---
+slug: policies
+append_help_link: true
+title: Policies
+hide_title: true
+description: The Policies page is a visual representation of the rules that Semgrep Secrets uses to scan code
+tags:
+  - Semgrep Secrets
+---
+
+# Policies
+
+The **Policies** page visually represents the rules Semgrep Secrets uses for scanning. 
+
+![Overview of Semgrep Secrets policies view](/img/secrets-rules-management.png)
+
+To access the policies page for Semgrep Secrets:
+
+1. Log in to Semgrep Cloud Platform and navigate to **Rules** > **Policies**.
+2. Switch to the **Secrets** page.
+
+## Policies page structure
+
+The Policies page consists of a header and three main panes:
+
+<dl>
+    <dt>Policies header</dt>
+        <dd>
+            The top header consists of:
+            <ul>
+             <li><strong>Validation State Policies</strong>, which lets you define how Semgrep handles findings that it categorizes as invalid or results in a validation error</li>
+                    <li><i class="fa-solid fa-gear"></i> <b>Rule Modes</b> button where you can view <strong>rule modes</strong> and edit <strong>notifications</strong> for each rule mode. Rule modes define what <strong>workflow actions</strong> Semgrep Secrets performs when a rule detects a finding. For example, setting a rule's mode to <strong>Comment</strong> means that Semgrep posts PR or MR comments from findings generated by that rule. See <a href="#validation-state-policies">Validation state policies</a> for more information.</li>
```

Comment:
Let's comment this out since this is unsupported at this time. We temporarily removed rule modes from the UI.